### DISCUSSION TITLE
color token reference

### DISCUSSION THREAD

**Discussion by vkoopmans-resoluut** [six7/figma-tokens#268] [Q&A] [answered]
I can't seem to get my color reference right.
Does anyone see what I'm doing wrong?

<img width="269" alt="Screenshot 2021-10-11 at 11 54 18" src="https://user-images.githubusercontent.com/40267611/136771253-923a4d13-c1f5-4cd6-bd84-3d93ba8a14ba.png">

<img width="339" alt="image" src="https://user-images.githubusercontent.com/40267611/136771358-46771fb8-3ca0-4231-87a5-59f280b68ca1.png">

**Comment by six7** [comment, marked as answer]
Your token is actually called `brand.primary` - not `colors.brand.primary` - change your alias to `$brand.primary`

Always insert the name in the name field, by default the plugin doesnt prepend a token type to a name

**Comment by vkoopmans-resoluut** [comment]
Ahh thank you, it had me confused 😁
Working now! Thanks 🙏 

__________
